euphony/src/main/java/co/euphony/ui/EuphonyTxPanel.kt

```diff
@@ -0,0 +1,101 @@
+package co.euphony.ui
+
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxHeight
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.*
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.Close
+import androidx.compose.material.icons.filled.Send
+import androidx.compose.runtime.*
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.Color.Companion.Black
+import androidx.compose.ui.graphics.Color.Companion.LightGray
+import androidx.compose.ui.graphics.Color.Companion.White
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.dp
+import androidx.lifecycle.viewmodel.compose.viewModel
+import co.euphony.common.Constants.*
+import co.euphony.tx.EuTxManager
+import co.euphony.ui.theme.LightBlue
+import co.euphony.ui.theme.LightSkyBlue
+import co.euphony.ui.viewmodel.TxPanelViewModel
+import co.euphony.ui.viewmodel.TxPanelViewModelFactory
+
+@Composable
+fun EuphonyTxPanel(
+    hint: String = "",
+    textColor: Color = Black,
+    hintTextColor: Color = LightGray,
+    textFieldBackgroundColor: Color = LightSkyBlue,
+    buttonBackgroundColor: Color = LightBlue,
+    panelHeight: Dp = 54.dp,
+    cornerRadius: Dp = 8.dp,
+    iconTintColor: Color = White
+) {
+    // TODO : EuTxManager() should be changed to use singleton pattern
+    val viewModel: TxPanelViewModel = viewModel(factory = TxPanelViewModelFactory(EuTxManager()))
+
+    val btnStatus = viewModel.isProcessing.collectAsState()
+    var textData by remember { mutableStateOf("") }
+
+    Row(
+        modifier = Modifier
+            .fillMaxWidth()
+            .height(panelHeight),
+        verticalAlignment = Alignment.CenterVertically
+    ) {
+        TextField(
+            modifier = Modifier
+                .weight(1f)
+                .fillMaxHeight(),
```

Comment:
I tested your `EuphonyTxPanel`, and even if row has `panelHeight`, your `TextField` and `Button` has `fillMaxHeight()`, so it fills whole screen. So, I think `TextField` and `Button` should have panelHeight.

Reply:
Thank you for letting me know! I fixed it
[15c0d37](https://github.com/euphony-io/euphony/pull/262/commits/15c0d37cbed82765754fa82a5751a96f574f9c15)